sdk/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpansProcessor.java

```diff
@@ -48,9 +43,6 @@ public void onStart(ReadableSpan span) {
 
   @Override
   public void onEnd(ReadableSpan span) {
-    if (!span.getSpanContext().getTraceOptions().isSampled()) {
```

Comment:
We should make this optional with a default to only export the isSampled.

Reply:
Do you want to add a parameter to constructor/builder?

Reply:
Yes, option on the Builder is good. Thanks

Reply:
Can you please fix this?

Reply:
Yes, sorry for the delay. I will submit a fix today